banzai_floyds/tests/utils.py

```diff
@@ -254,6 +248,23 @@ def generate_fake_extracted_frame(do_telluric=False, do_sensitivity=True):
     return frame
 
 
+def load_manual_region(region_filename, site_id, order_id, shape, order_height):
```

Comment:
This is a utility function for loading the results of the manual order center fits in the e2e tests.